src/components/mgt-person/mgt-person.scss

```
@@ -91,6 +89,86 @@ mgt-person .user-avatar {
   img {
     height: 100%;
   }
+
+  &.pinkRed10 {
+    background-color: $ms-color-sharedPinkRed10;
```

Keep the existing css custom property and remove the component attribute.

Do this for all colors

```suggestion
    background-color: var(--initials-background-color, $ms-color-sharedPinkRed10);
```

---

Updated